### ISSUE TITLE
Transition animation sometimes does not work when scrollen too far

### ISSUE THREAD

**Issue by carstenhag** [haroldadmin/MovieDB#3]
I hope the video can help you understand what I'm saying :) 

https://drive.google.com/file/d/1lwCxMOMJCrlhmGP5ZMlf0IQ_Odms137t/view?usp=drivesdk

**Comment by haroldadmin** [comment]
Yes, I know this is an annoying thing. I don't know how to solve this yet, do you have any ideas in mind?

**Comment by carstenhag** [comment]
Sadly no :( I have just worked with these sort of transitions once, not
experienced at all with them yet.

Kshitij Chauhan <notifications@github.com> schrieb am Sa., 2. März 2019,
12:56:

**Comment by haroldadmin** [comment]
Okay, no problem. I have some guesses as to what I can do to fix this.
